style-packages.md

```diff
@@ -0,0 +1,255 @@
++++
+date        = "2017-01-14T11:27:27-04:00"
+title       = "Style guideline for Go packages"
+description = "Best practices to organize and name Go packages"
+tags        = [ "readability" ]
+type        = "archived"
+draft       = false
++++
+
```

Comment:
I feel like there's 4 main sections.
* Package Naming
* Package Organization
* Package Documentation
* Project Organization

Above those is the introduction & what is a package.